tests/test_eslog.py

```diff
@@ -0,0 +1,50 @@
+from nose.tools import eq_
+from . import TestGramex
+import json
+import time
+
+
+class TestESLog(TestGramex):
+    def test_eslog(self):
+        self.check('/log', {'level': 'INFO', 'x': 1, 'msg': 'abc'})
+        result = self.check('/log/queue')
+        result = json.loads(result.content)
+        eq_(result[0]['level'], 'INFO')
+        eq_(result[0]['x'], '1')
+        eq_(result[0]['msg'], 'abc')
+        eq_(result[0]['port'], 9999)
+        # TODO: Check that this is within 5 seconds of now
+        # eq_(result['time'], )
+
+        # This logs ``{level: INFO, x: 1, msg: abc, port: 9988, time: 2020-07-21 13:41:00}``.
+        # 3 keys are added:
+
+        # 1. ``level``: logging level. Defaults to INFO
+        # 2. ``time``: current time as YYYY-MM-DD HH:MM:ZZ in UTC
+        # 3. ``port``: application's current port
+
+        # If a logging service like ElasticSearch has been configured, it will periodically flush
+        # the logs into ElasticSearch.
+
+        # Run without configuring gramexlog
+        # Run after configuring gramexlog manually
+
+        # Without eslog config, log queue should be empty
+        # Check if the log is written to elastic search
+
+    def test_eslog_not_running(self):
+        # 1. Create a connection to a nonexistent elasticsearch port and send the request
+        # 2. Stop elasticsearch and send a log message
+        # self.check('/log', {'level': 'WARN', 'x': x, 'msg': 'abc'})
+        pass
+
```

Comment:
Add a test case to send log messages to `default` and to `app` and ensure that they go to the right index.